docs/upgrade-guides/1.x-to-2.0.0.md

```diff
@@ -0,0 +1,74 @@
+## Upgrade from Radical Presentation 1.x to Radical 2.0.0 for .NET Core
+
+### Main facts
+
+Starting with V2 radical Framework supports only .NET Core 3, the `Radical` package targets `netstandard2.0` and the `Radical.Windows` package targets `netcoreapp3.0`. WPF .NET Framework (what is generally referred as full framework) projects are not supported.
+
+The `Radical.Windows.Presentation` package has been merged into `Radical.Windows` and by default the default IoC container provided by `Microsoft.Extensions.DependencyInjection` will be used. V2.0.0 doesn't support external DI/IoC containers, external containers will be supported in a future minor along with generic host support.
```

Comment:
considering workitem "third-party containers (# 113)" this content should be updated